pkg/trace/test/testsuite/traces_test.go

```diff
@@ -87,6 +87,30 @@ func TestTraces(t *testing.T) {
 			payloadsEqual(t, append(p[:2], p[3:]...), v)
 		})
 	})
+
+	t.Run("filter_tags", func(t *testing.T) {
+		if err := r.RunAgent([]byte("apm_config:\r\n  filter_tags:\r\n require: "["env:prod", db:mysql"]" \r\n"reject": ["outcome:"success"]}]")); err != nil {
```

Comment:
The reject part is irrelevant, so we may remove it.
```suggestion
		if err := r.RunAgent([]byte("apm_config:\r\n  filter_tags:\r\n require: "["env:prod", db:mysql"]"")); err != nil {
```